Sources/NetworkProtection/Repositories/NetworkProtectionLocationListRepository.swift

```diff
@@ -21,9 +21,11 @@ import Common
 
 public protocol NetworkProtectionLocationListRepository {
     func fetchLocationList() async throws -> [NetworkProtectionLocation]
+    func fetchLocationListIgnoringCache() async throws -> [NetworkProtectionLocation]
```

Comment:
Suggestion: For readability, implement a `fetchLocationList(cachePolicy:)` method instead of having 2 separate ones.

Or, at least comment on how these 2 methods are used.
- `fetchLocationList()`: Display the location list on the location selector
- `fetchLocationListIgnoringCache()`: Resolve the location using fallback logic when generating tunnel config